package.json

```diff
@@ -15,7 +15,7 @@
     "@codemirror/state": "6.3.3",
     "@codemirror/view": "^6.22.2",
     "@lezer/highlight": "^1.2.0",
-    "@redocly/realm": "^0.78.6",
+    "@redocly/realm": "^0.78.8",
```

Comment:
You can drop the `package.json` and `package-lock.json` changes from this PR; we've already updated to a newer version than this in the `master` branch.